03_letters/jacklee/letter.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	m := map[rune]int{}
+	for _, v := range s {
+		m[v]++
+	}
+	return m
+}
+
+func sortLetters(m map[rune]int) []string {
+	l := len(m)
+	keys := make([]int, 0, l)
```

Comment:
I was going to ask you to change this to use a fixed size array, populated directly with an index variable, then did some research and found out this method isn't much slower (benchmarks ~20% slower, pretty sure no resizing is going on). Nice work on finding this method 👍. I'll leave it up to you whether you want to change this to gain that 20% speedup.